extensions/localserver-cli-command/src/module-loader/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I don't see the package-lock.json that corresponds to this file.
Is it intended not to have it?